src/content/impact/design-for-sustainability/dfs.scss

```diff
@@ -0,0 +1,134 @@
+.dfs-header-bottom {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+
+    @include breakpoint('sm') {
+        position: relative;
+        padding-bottom: 2rem;
+    }
+
+    .carbon--button {
+        min-height: 3rem;
+        min-width: 13.75rem;
+        text-align: left;
+        background-color: #0062ff;
+        border: 0;
+        color: #fff;
+        font-size: 16px;
+        padding-left: 1rem;
+        font-weight: 400;
+        -webkit-font-smoothing: antialiased;
+        text-rendering: optimizeLegibility;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 0 1rem;
+        cursor: pointer !important;
+
+        @include breakpoint('sm') {
+            min-width: 100%;
+        }
+
+        @include breakpoint('md') {
+            min-width: 13.75rem;
+        }
+    }
+
+    .carbon--button:hover {
+        background-color: #0050e6;
+    }
+
+    .carbon--button:active {
+        background-color: #002d9c;
+    }
+
+    .carbon--button:focus {
+        box-shadow: inset 0 0 0 1px #0f62fe ,inset 0 0 0 2px #f4f4f4;
+        border-color: #0f62fe;
+    }
+
+    .hero--subtext {
+        font-size: calc(1.25rem + .25 * ((100vw - 82rem)/ 17));
+        color: #525252;
+        padding-bottom: 2.5rem;
+        line-height: 1.75rem;
+        
+        @include breakpoint('md') {
+            font-size: 1.25rem;
+            line-height: auto;
+        }
+        
+        @include breakpoint('sm') {
+            font-size: calc(1.25rem + 0 * ((100vw - 20rem) / 62));
+            line-height: auto;
+            padding-bottom: 1rem;
+        }
+    }
+}
+
+.dfs-header-top {
+    position: absolute;
+    top: 0;
+
+    @include breakpoint('sm') {
+        position: relative; 
+    }
+    
+    h1 {
+        font-size: calc(3rem + .75 * ((100vw - 82rem)/ 17));
+
+        @include breakpoint('sm') {
+            font-size:  calc(2.625rem + 0 * ((100vw - 20rem) / 22));
+            line-height: auto;
+            padding-bottom: 1rem;
+        }
+    }
+}
+
+.background {
+    @include breakpoint('sm') {
+        padding-bottom: 2rem !important;
+        
+    }
+}
+
+.tile--feature .bx--tile {
+    @media screen and (max-width: 671px) {
+        position: relative !important;
+        width: 100% !important;
+        max-height: 96px !important;
+    }
+}
+
+.tile__link-icon {
+    @media screen and (max-width: 671px) {
+        position: absolute !important;
+        right: 1rem !important;
+        bottom: initial !important;
+    }
+
+    @media screen and (min-width: 672px) {
+        right: 1rem !important;
+        top: 5rem;
+    }
+
+    @media screen and (min-width: 1055px) {
+        right: 1rem !important;
+        top: 5rem;
+    }
+}
+
+.tile--feature .bx--tile {
+    @media screen and (min-width: 672px){
+        height: 50%;
+    }
+
+    @media screen and (min-width: 1055px) {
+        height: initial;
+    }
+}
```

Comment:
These look like pretty generic classes that are getting declared for the page, which might affect other pages. Would you be able to nest in a page level class to be safe?